getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -27,58 +27,51 @@ To create a Telerik-enabled Blazor project:
 
-The **Telerik UI for Blazor Template Wizard** provides two options to start you project:
+> The **Admin**, **Dashboard**, **CRUD**, **Form**, and **Chart** templates were deprecated. Their source code is available in the [`blazor-ui` repo](https://github.com/telerik/blazor-ui/tree/master/common/legacy-project-templates).
 
-* **Start from Blank App**&mdash;allows you to create a blank project that is pre-configured for the Progress® Telerik® UI for Blazor components. You can select and add sample pages to get started (a Grid with CRUD operations form a service, a Form, Charts).
+The wizard contains the following sections:
 
-* **Start from Template** - the wizard provides a set of predefined project templates:
-    * **CRUD, Form, Chart**&mdash;A small app that showcases the Telerik UI Data Grid, Chart, and Form validation. The Data Grid also uses a basic CRUD service (a common pattern for data updates in production-ready applications).
-    * **Dashboard**&mdash;A basic dashboard that showcases the Telerik UI TileLayout component and how individual blocks (tiles) can fetch data.
-    * **Admin**&mdash;A small app that showcases a dashboard for administrators. The app uses some of the main Telerik UI components like the [Data Grid](slug:grid-overview), [Chart](slug:components/chart/overview), [TileLayout](slug:tilelayout-overview), [Form](slug:form-overview), and more.
-
-The available templates vary depending on the target framework and hosting model. All of the above templates are available for the Client and Server projects for .NET 6 and .NET 7. The Hybrid projects and the .NET 8 Web App support only the **Start from Blank App** option.
-
-#### Section 2: Select Project Name and Path
+#### Section 1: Select Project Name and Path
 
 Type your project name and select the output folder for it.
 
-#### Section 3: Install or Update License Key
+#### Section 2: Install or Update License Key
 
 If necessary, the Telerik Visual Studio Code extension will notify you about a missing or outdated [license key](slug:installation-license-key), and will install or update it.
 
-#### Section 4: Select UI for Blazor Version
+#### Section 3: Select UI for Blazor Version
 
 Select the preferred UI for Blazor version. Make sure that your system has access to the specified version of the UI components. You can achieve this with the [Telerik NuGet feed](slug:installation/nuget).
 
-#### Section 5: Select Hosting Model
+#### Section 4: Select Hosting Model
 
 Here you can select the desired hosting model&mdash;[**WebApp**](slug:getting-started/web-app), [**Client** (also: WebAssembly)](slug:getting-started/client-side), [**Server**](slug:getting-started/server-side), or [**Hybrid** (a hybrid Blazor MAUI)](slug:getting-started/hybrid-blazor).
 
 The available target framework options are updated upon selecting the hosting model.
 
-#### Section 6: Select Your License Type
+#### Section 5: Select Your License Type
```

Comment:
Just a side note. It is a pretty weird workflow to install/update the license key in section 2 but select license type in section 5.

Reply:
Sections in this article are listed following their order in the extension. That said, we may share this feedback with the VSX team.